circle.yml

```diff
@@ -19,7 +19,7 @@ checkout:
   post:
     - git pull origin ${CIRCLE_BRANCH}
     - git clone git@github.com:wholesale-design-system/styleguide.git
-    - git config --global user.email circleci@circleci
+    - git config --global user.email t.schlage@gmx.net
```

Comment:
I would say, in the future something like "tech-fabric@zalando.de" where all engineers working with it are subscribed ^^